supply/lib/supply/client.rb

```diff
@@ -544,29 +549,14 @@ def fetch_images(image_type: nil, language: nil)
         )
       end
 
-      urls = (result.images || []).map(&:url)
-      images = urls.map do |url|
-        uri = URI.parse(url)
-        clean_url = [
-          uri.scheme,
-          uri.userinfo,
-          uri.host,
-          uri.port,
-          uri.path
-        ].join
-
-        UI.verbose("Initial URL received: '#{url}'")
-        UI.verbose("Removed params ('#{uri.query}') from the URL")
-        UI.verbose("URL after removing params: '#{clean_url}'")
```

Comment:
All those lines were dead code even before my change: as you can see, `clean_url` was never used in the returned value after all.